packages/ui/src/theme/css/component/switchField.scss

```diff
@@ -0,0 +1,70 @@
+.amplify-switchfield {
+  display: inline-flex;
+  align-items: center;
+  &[data-size='small'] {
+    font-size: var(--amplify-components-switchfield-small-font-size);
+  }
+  &[data-size='large'] {
+    font-size: var(--amplify-components-switchfield-large-font-size);
+  }
+  &[data-label-position='start'] {
+    flex-direction: row;
+  }
+  &[data-label-position='end'] {
+    flex-direction: row-reverse;
+  }
+  &[data-label-position='top'] {
+    flex-direction: column;
+  }
+  &[data-label-position='bottom'] {
+    flex-direction: column-reverse;
+  }
+}
```

Comment:
I think there needs to be more padding between the label and the switch itself. Not sure where to comment on that, just noticed it from the docs.
<img width="156" alt="switch" src="https://user-images.githubusercontent.com/6165315/131931515-0d1a0fa4-bbf1-458f-9f99-229867eb7677.png">

Reply:
more padding has been added